tests/contrib/langchain/conftest.py

```diff
@@ -72,6 +72,16 @@ def mock_tracer(langchain, mock_logs, mock_metrics):
     mock_metrics.reset_mock()
 
 
+@pytest.fixture
+def mock_llmobs_writer():
+    patcher = mock.patch("ddtrace.llmobs._llmobs.LLMObsWriter")
+    LLMObsWriterMock = patcher.start()
+    m = mock.MagicMock()
+    LLMObsWriterMock.return_value = m
+    yield m
+    patcher.stop()
```

Comment:
We should wrap this in a `try-finally`. We should always stop the patcher even if the test fails unexpectedly.